modules/afsocket/socket-options.c

```diff
@@ -49,6 +49,20 @@ socket_options_setup_socket_method(SocketOptions *self, gint fd, GSockAddr *bind
                           evt_tag_int("so_rcvbuf_set", so_rcvbuf_set));
             }
         }
+      if (self->so_reuseport)
+        {
+#ifdef SO_REUSEPORT
+          if (setsockopt(fd, SOL_SOCKET, SO_REUSEPORT, &self->so_reuseport, sizeof(self->so_reuseport)) < 0)
+            {
+              msg_error("The kernel refused our SO_REUSEPORT setting, which should be supported by Linux 3.9+",
+                        evt_tag_error("error"));
+              return FALSE;
+            }
+#else
+          msg_error("You set so-reuseport() to 1, but your platform does not support SO_REUSEPORT, which is supported on Linux 3.9+");
```

Comment:
`so-reuseport()` is a yes/no option. Should not we rephrase this error message? For example: "You set  so-reuseport() to yes ..." or "You enabled so-reuseport(), but ..."

Reply:
I clarified the error message, thanks for pointing that out.